src/learn.jl

```diff
@@ -292,12 +291,14 @@ Where...
 
 - `classes` all possible classes voted on.
 
-Returns `nothing` if `votes` has only a single voter (i.e., a single column) or if
+Returns `(per_class_IRA_kappas=missing, multiclass_IRA_kappas=missing)` if `votes` has only a single voter (i.e., a single column) or if
 no two voters rated the same sample. Note that vote entries of `0` are taken to
 mean that the voter did not rate that sample.
 """
 function _calculate_ira_kappas(votes, classes)
-    (isnothing(votes) || size(votes, 2) < 2) && return nothing  # no votes given or only one expert
+    # no votes given or only one expert:
+    (isnothing(votes) || size(votes, 2) < 2) &&
+        return (; per_class_IRA_kappas=missing, multiclass_IRA_kappas=missing)
```

Comment:
in this case, still okay to return `missing` not `NaN` b/c we aren't serializing these as part of a vector---the schema accounts for them